<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Software</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Palash Sashittal</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-item"><a href="software.html" class="current">Software</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Software</h1>
</div>
<h2>Cell lineage and differentiation</h2>
<h3><a href="https://github.com/raphael-group/MERLIN" target=&ldquo;blank&rdquo;>Merlin</a></h3>
<p>Merlin uses a novel nested perfect phylogeny model to jointly infer cell lineage trees and mitochondrial clone tree using mitochondrial muations detected from single cell sequencing data. [<a href="https://github.com/raphael-group/MERLIN" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b>, Viola Chen, Amey Pasarkar, Benjamin Raphael, <a href="https://academic.oup.com/bioinformatics/article/40/Supplement_1/i218/7700844" target=&ldquo;blank&rdquo;>Joint inference of cell lineage and mitochondrial evolution from single-cell sequencing data</a>, <i>Bioinformatics</i>, 2024.
</p>
</li>
</ul>
<h3><a href="https://github.com/raphael-group/startle" target=&ldquo;blank&rdquo;>Startle</a></h3>
<p>Startle uses the maximum parsimony framework to find single-cell lineage trees from CRISPR-Cas9 based lineage tracing data using the star homoplasy model. [<a href="https://github.com/raphael-group/startle" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b>*, Henri Schmidt*, Michelle Chan and Benjamin Raphael, <a href="https://www.cell.com/cell-systems/abstract/S2405-4712(23)00328-9" target=&ldquo;blank&rdquo;>Startle: a star homoplasy approach for CRISPR-Cas9 lineage tracing</a>, <i>Cell Systems</i>, 2023.
</p>
</li>
</ul>
<h2>Cancer genome evolution</h2>
<h3><a href="https://github.com/raphael-group/constrained-Dollo" target=&ldquo;blank&rdquo;>ConDoR</a></h3>
<p>ConDoR is an algorithm to infer tumor phylogenies from targeted single-cell DNA sequencing data using the Constrained Dollo model. [<a href="https://github.com/raphael-group/constrained-Dollo" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b>, Hoachen Zhang, Christine Iacobuzio-Donahue and Benjamin Raphael, <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03106-5" target=&ldquo;blank&rdquo;>ConDoR: Tumor phylogeny inference with a copy-number constrained mutation loss model</a>, <i>Genome Biology</i>, 2023.
</p>
</li>
</ul>
<h3><a href="https://github.com/raphael-group/lazac-copy-number" target=&ldquo;blank&rdquo;>Lazac</a></h3>
<p>Lazac is a tool for inferring single-cell resolution copy number phylogenies from copy number data. It is based on our zero-agnostic copy number transformation (ZCNT) model, which we recently introduced as a simplification of the copy number transformation (CNT) model. [<a href="https://github.com/raphael-group/lazac-copy-number" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p>Henri Schmidt, <b>Palash Sashittal</b> and Benjamin Raphael, <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011590" target=&ldquo;blank&rdquo;>A zero-agnostic model for copy number evolution in cancer</a>, <i>PLOS Computational Biology</i>, 2023.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/paction" target=&ldquo;blank&rdquo;>Paction</a></h3>
<p>Paction is a method to reconcile SNV and CNA clone proportions and trees for thesame tumor. The output is a set of tumor clones characterized by both SNVs and CNAs,the clone proportions and a comprehensive clone tree annotated with both SNV andCNA events.
[<a href="https://github.com/elkebir-group/paction" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b>, Simone Zaccaria and Mohammed El-Kebir, <a href="https://almob.biomedcentral.com/articles/10.1186/s13015-022-00209-9" target=&ldquo;blank&rdquo;>Parsimonious clone tree integration in cancer</a>, <i>Algorithms and Molecular Biology</i>, 2022.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/doubletD" target=&ldquo;blank&rdquo;>doubletD</a></h3>
<p>doubletD is an efficient doublet detector designed for single-cell DNA sequencing (scDNA-seq) data. Incorporating doubletD in scDNA-seq analysis pipelines leads to more accurate cancer genotyping and tumor phylogeny inference.
[<a href="https://github.com/elkebir-group/doubletD" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p>Leah Weber*, <b>Palash Sashittal</b>*, and Mohammed El-Kebir, <a href="https://academic.oup.com/bioinformatics/article/37/Supplement_1/i214/6319699?login=false" target=&ldquo;blank&rdquo;>doubletD: detecting doublets in single-cell DNA sequencing data</a>, <i>Bioinformatics</i>, 2021.
</p>
</li>
</ul>
<h2>Infectious disease evolution and transmission</h2>
<h3><a href="https://github.com/elkebir-group/primes" target=&ldquo;blank&rdquo;>PRIMES</a></h3>
<p>PRIMES is a computational to predict the sensitivity and specificity of a PCR assay to detect specific pathogen lineages of interest.[<a href="https://github.com/elkebir-group/primes" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p>Chamtuet Oh*, <b>Palash Sashittal</b>*, Aijia Zhou, Leyi Wang, Mohammed El-Kebir and Thanh Nguyen,<a href="https://journals.asm.org/doi/full/10.1128/aem.02289-21" target=&ldquo;blank&rdquo;>Design of SARS-CoV-2 variant-specific PCR assays considering regional and temporal characteristics</a>, <i>Applied and Environmental Microbiology</i>, 2022.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/CORSID" target=&ldquo;blank&rdquo;>CORSID</a></h3>
<p>CORSID is an algorithm to simultaneously identify TRS sites, the core sequence and gene locations given an unannotated coronavirus genome sequence. [<a href="https://github.com/elkebir-group/CORSID" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p>Chuanyi Zhang*, <b>Palash Sashittal</b>*, Michael Xiang, Yichi Zhang, Ayesha Kazi and Mohammed El-Kebir, <a href="https://academic.oup.com/mbe/article/39/7/msac133/6608352" target=&ldquo;blank&rdquo;>Accurate identification of transcription regulatory sequences and genes in Coronaviruses</a>, <i>Molecular Biology and Evolution</i>, 2022.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/Jumper" target=&ldquo;blank&rdquo;>Jumper</a></h3>
<p>Jumper is a code that takes a bam file of RNA-seq reads and a Fasta file of the reference genome as input and reconstructs RNA transcripts and their abundances in the sample.
[<a href="https://github.com/elkebir-group/Jumper" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b>, Chuanyi Zhang, Jian Peng and Mohammed El-Kebir, <a href="https://www.nature.com/articles/s41467-021-26944-y" target=&ldquo;blank&rdquo;>Jumper enables discontinuous transcript assembly in Coronaviruses</a>, <i>Nature Communications</i>, 2021.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/TiTUS" target=&ldquo;blank&rdquo;>TiTUS</a></h3>
<p>TiTUS is a code that takes as input a timed pathogen phylogeny with leaves labelled by the host along with epidemiological data such as entry-removal times for the hosts and a contact map.
It counts and uniformly samples from the set of feasbile interval vertex labelings of the timed phylogeny that satisfy the direct transmission constraint while supporting a weak transmission bottleneck.
[<a href="https://github.com/elkebir-group/TiTUS" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Palash Sashittal</b> and Mohammed El-Kebir, <a href="https://academic.oup.com/bioinformatics/article/36/Supplement_1/i362/5870507" target=&ldquo;blank&rdquo;>Sampling and summarizing transmission trees with multi-strain infections</a>, <i>Bioinformatics</i>, 2020.
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/SharpTNI" target=&ldquo;blank&rdquo;>SharpTNI</a></h3>
<p>SharpTNI is a tool for counting and sampling solutions from the space of parsimonious transmission networks under a weak transmission bottleneck constraint.
This problem arises in phylodynamic and phylogeographic analyses of disease outbreaks.
[<a href="https://github.com/elkebir-group/SharpTNI" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; El-Kebir, M., <a href="https://www.biorxiv.org/content/10.1101/842237v1" target=&ldquo;blank&rdquo;>SharpTNI:  Counting and Sampling Parsimonious Transmission Networks under a Weak Bottleneck</a>, <i>BMC Medical Genomics</i>, 2020. [<a href="https://uofi.box.com/s/3b6bawcwl6qb951hbvr1rmyop250okrq" target=&ldquo;blank&rdquo;>Paper</a>]
</p>
</li>
</ul>
<h2>Data-driven modeling of flow dynamics</h2>
<h3><a href="https://bitbucket.org/sashitt2/lrdmd/src/master" target=&ldquo;blank&rdquo;>lrDMD</a></h3>
<p>Low-rank dynamic mode decomposition (lrDMD) solves for a rank-constrained linear representation of the dynamical system for given series of data snapshots.
[<a href="https://bitbucket.org/sashitt2/lrdmd/src/master" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; Bodony, D., <a href="https://link.springer.com/article/10.1007/s00162-019-00508-9" target=&ldquo;blank&rdquo;>Reduced-Order Control using Low-Rank Dynamic Mode Decomposition</a>, <i>Theoretical and Computational Fluid Dynamics</i>, 2019. [<a href="https://uofi.box.com/s/clgq0t1y0o3iyopkq0b2qcti5c2nvdvc" target=&ldquo;blank&rdquo;>Paper</a>]
</p>
</li>
</ul>
<h3><a href="https://github.com/sashitt2/optimal_sensing" target=&ldquo;blank&rdquo;>KF-DMD</a></h3>
<p>A data-driven method for optimal sensor placement and parameter optimization using transfer operator based reduced order models.
[<a href="https://github.com/sashitt2/optimal_sensing" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; Bodony, D., <a href="https://link.springer.com/article/10.1007/s00162-021-00584-w" target=&ldquo;blank&rdquo;>Data-driven sensor placement for fluid flows</a>, <i>Theoretical and Computational Fluid Dynamics</i>, 2021.
</p>
</li>
</ul>
</td>
</tr>
</table>
</body>
</html>
