<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Software</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Palash Sashittal</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="software.html" class="current">Software</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Software</h1>
</div>
<h2>Cancer Phylogenetics</h2>
<h3><a href="https://github.com/elkebir-group/paction" target=&ldquo;blank&rdquo;>Paction</a></h3>
<p>Paction is a method to reconcile SNV and CNA clone proportions and trees for thesame tumor. The output is a set of tumor clones characterized by both SNVs and CNAs,the clone proportions and a comprehensive clone tree annotated with both SNV andCNA events.
[<a href="https://github.com/elkebir-group/paction" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<h2>Doublet Detection</h2>
<h3><a href="https://github.com/elkebir-group/doubletD" target=&ldquo;blank&rdquo;>doubletD</a></h3>
<p>doubletD is an efficient doublet detector designed for scDNA-seq data. IncorporatingdoubletD in scDNA-seq analysis pipelines leads to more accurate results.
[<a href="https://github.com/elkebir-group/doubletD" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<h2>Transcript Assembly</h2>
<h3><a href="https://github.com/elkebir-group/Jumper" target=&ldquo;blank&rdquo;>Jumper</a></h3>
<p>Jumper is a code that takes a bam file of RNA-seq reads and a Fasta file of the reference genome as input and reconstructs RNA transcripts and their abundances in the sample.
[<a href="https://github.com/elkebir-group/Jumper" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<h2>Phylodynamics</h2>
<h3><a href="https://github.com/elkebir-group/TiTUS" target=&ldquo;blank&rdquo;>TiTUS</a></h3>
<p>TiTUS is a code that takes as input a timed pathogen phylogeny with leaves labelled by the host along with epidemiological data such as entry-removal times for the hosts and a contact map.
It counts and uniformly samples from the set of feasbile interval vertex labelings of the timed phylogeny that satisfy the direct transmission constraint while supporting a weak transmission bottleneck.
[<a href="https://github.com/elkebir-group/TiTUS" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; El-Kebir, M., <a href="https://academic.oup.com/bioinformatics/article/36/Supplement_1/i362/5870507" target=&ldquo;blank&rdquo;>Sampling and Summarizing Transmission Trees with Multi-strain Infections</a>, <i>Bioinformatics</i>, 2020. [<a href="https://uofi.box.com/s/q5uabj764ixjoi9zjqdafubp9pi1bksz" target=&ldquo;blank&rdquo;>Paper</a>]
</p>
</li>
</ul>
<h3><a href="https://github.com/elkebir-group/SharpTNI" target=&ldquo;blank&rdquo;>SharpTNI</a></h3>
<p>SharpTNI is a tool for counting and sampling solutions from the space of parsimonious transmission networks under a weak transmission bottleneck constraint.
This problem arises in phylodynamic and phylogeographic analyses of disease outbreaks.
[<a href="https://github.com/elkebir-group/SharpTNI" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; El-Kebir, M., <a href="https://www.biorxiv.org/content/10.1101/842237v1" target=&ldquo;blank&rdquo;>SharpTNI:  Counting and Sampling Parsimonious Transmission Networks under a Weak Bottleneck</a>, <i>BMC Medical Genomics</i>, 2020. [<a href="https://uofi.box.com/s/3b6bawcwl6qb951hbvr1rmyop250okrq" target=&ldquo;blank&rdquo;>Paper</a>]
</p>
</li>
</ul>
<h2>Data-driven Dynamics Modeling</h2>
<h3><a href="https://bitbucket.org/sashitt2/lrdmd/src/master" target=&ldquo;blank&rdquo;>lrDMD</a></h3>
<p>Low-rank dynamic mode decomposition (lrDMD) solves for a rank-constrained linear representation of the dynamical system for given series of data snapshots.
[<a href="https://bitbucket.org/sashitt2/lrdmd/src/master" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; Bodony, D., <a href="https://link.springer.com/article/10.1007/s00162-019-00508-9" target=&ldquo;blank&rdquo;>Reduced-Order Control using Low-Rank Dynamic Mode Decomposition</a>, <i>Theoretical and Computational Fluid Dynamics</i>, 2019. [<a href="https://uofi.box.com/s/clgq0t1y0o3iyopkq0b2qcti5c2nvdvc" target=&ldquo;blank&rdquo;>Paper</a>]
</p>
</li>
</ul>
<h3><a href="https://github.com/sashitt2/optimal_sensing" target=&ldquo;blank&rdquo;>KF-DMD</a></h3>
<p>A data-driven method for optimal sensor placement and parameter optimization using transfer operator based reduced order models.
[<a href="https://github.com/sashitt2/optimal_sensing" target=&ldquo;blank&rdquo;>Code</a>]
</p>
<ul>
<li><p><b>Sashittal, P.</b> &amp; Bodony, D., Data-driven sensor placement for fluid flows, Under review for <i>Theoretical and Computational Fluid Dynamics</i>, 2021.
</p>
</li>
</ul>
</td>
</tr>
</table>
</body>
</html>
